File: 0001 Code Chef/0023 Chef Plays Ludo/implementation/0006 php/ludo.php
<?php
$test = (int) fgets(STDIN);
while($test > 0){
    $x = (int) fgets(STDIN);
    if($x == 6){
        echo "YES\n";
    }else{
        echo "NO\n";
    }
    $test = $test - 1;
}
?>

File: 0001 Code Chef/0023 Chef Plays Ludo/implementation/0001 c/ludo.c
#include <stdio.h>

int main(){
    int test;
    scanf("%d",&test);
    while(test > 0){
        int x;
        scanf("%d",&x);
        if(x == 6){
            printf("YES\n");
        }else{
            printf("NO\n");
        }
        test = test - 1;
    }
    return 0;
}